.github/workflows/cli.test.yaml

```
@@ -65,19 +65,21 @@ jobs:
           go mod tidy
 
       - name: Delegator db migrate
-        run: yarn delegator prisma-migrate dev --name init-delegator
-        env:
-          DATABASE_URL: "postgresql://postgres:postgres@localhost:5432/delegator?schema=public"
+        run: |
+          cd ./delegator
+          migrate -database "postgresql://postgres:postgres@localhost:5432/delegator?search_path=public&sslmode=disable" -verbose -path ./migrations up
```

No biggie, but it would be nicer with `working-directory`.

```suggestion
        working-directory: ./delegator
        run: migrate -database "postgresql://postgres:postgres@localhost:5432/delegator?search_path=public&sslmode=disable" -verbose -path ./migrations up
```

Same for run delegator below.